### ISSUE TITLE
Source code highlighter may produce escaped entities

### ISSUE THREAD

**Issue by pjanx** [bytesparadise/libasciidoc#968]
````
:source-highlighter: chroma
:unicode: false

```c
…
```
````

```
<div class="listingblock">
<div class="content">
<pre class="chroma highlight"><code data-lang="c"><span class="tok-o">&amp;</span><span class="tok-err">#</span><span class="tok-mi">8230</span><span class="tok-p">;</span></code></pre>
</div>
</div>
```

**Comment by xcoulon** [comment]
Thanks for reporting this, @pjanx!
(I really have a problem with encoding! 😅)

**Comment by xcoulon** [comment]
@pjanx I couldn't reproduce the problem after merging #969. 
Could you confirm it's all good for you now?

**Comment by pjanx** [comment]
@xcoulon don't worry, it's still there. :) But feel free to postpone it.

**Comment by xcoulon** [comment]
> @xcoulon it's still there, I've adjusted the reproducer. :) But feel free to postpone it.

Ah right, my bad, I can reproduce it now! (I was probably lacking coffee when I tried it yesterday morning 🤷‍♂️ )

**Comment by xcoulon** [comment]
fixed in #973 (wrong "Fixes" mention in the PR's commit msg 🤦‍♂️)
